website/versioned_docs/version-v0.7.x/quick-start.md

```diff
@@ -51,6 +51,12 @@ This guide illustrates how to patch outdated containers with `copa`.
 
         :::tip 
         If you want to patch an image using the digest, run the following command instead: 
+    
+        ```bash
```

Comment:
we shouldn't change it in version 0.7.0 docs since this will be a part of the next release